packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -20,7 +20,19 @@
   }
 
   [part='rephrase-button'] {
-    @apply rephrase-with-text;
+    @apply rephrase-with-text flex-auto justify-center;
+
+    .icon-container {
+      @apply mx-0;
```

Comment:
Same as above

Reply:
same here defined for mixin mobile layout